index.html

```diff
@@ -1,21 +1,428 @@
+      <b>
+        <a class="nav-link text-body" href="#">World</a>
+      </b>
+    </li>
+    <li class="nav-item border-right">
+      <b>
+        <a class="nav-link text-body" href="#">BUSINESS</a>
+      </b>
+    </li>
+    <li class="nav-item border-right">
+      <b>
+        <a class="nav-link text-body" href="#">TECH n SCIENCE</a>
+      </b>
+    </li>
+    <li class="nav-item border-right">
+      <b>
+        <a class="nav-link text-body" href="#">CULTURE</a>
+      </b>
+    </li>
+    <li class="nav-item border-right">
+      <b>
+        <a class="nav-link text-body" href="#">SPORT</a>
+      </b>
+    </li>
+    <li class="nav-item">
+      <b>
+        <a class="nav-link text-body" href="#">HEALTH</a>
+      </b>
+    </li>
+  </ul>
+
+  <main class="container-fluid mt-3">
+    <div class="row">
+      <div class="col-md-8">
+        <div class="container-fluid">
+          <div class="row">
+            <div class="col-md-5 ">
+              <div class="featured-stories ">
+                <h5 class="text-left text-secondary h6 font-weight-bold">FEATURED STORIES</h5>
+                <div class=" border-0">
+                  <img src="https://s.newsweek.com/sites/www.newsweek.com/files/styles/sm/public/2019/03/13/gettyimages-831982426.jpg"
+                    class="card-img-top" alt="...">
+                  <div class="card-body">
+                    <h5 class="card-title">GOP Senators Roll Out Bill to Curb Emergency Powers</h5>
+                    <p class="card-text">"Congress gave these legislative powers away in 1976, and it is far past
+                      time that
+                      we as an
+                      institution took them back," said Senator Mike Lee, a Utah Republican.</p>
+                    <hr>
+                    <h6>Trump’s Climate Denial Gets More Ridiculous By The Day</h6>
+                    <hr>
+                  </div>
+                </div>
+              </div>
+
+              <div class="featured-stories ">
+                <div class="card mb-0 border-0">
+                  <img src="https://s.newsweek.com/sites/www.newsweek.com/files/styles/sm/public/2019/03/13/ivanka-trump-and-senior-advisor-jared-kushner.jpg"
+                    class="card-img-top" alt="...">
+                  <div class="card-body">
+                    <h5 class="card-title">GOP Senators Roll Out Bill to Curb Emergency Powers</h5>
+                    <p class="card-text">"Congress gave these legislative powers away in 1976, and it is far past time
+                      that
+                      we as an
+                      institution took them back," said Senator Mike Lee, a Utah Republican.</p>
+                    <hr>
+                    <h6>Trump’s Climate Denial Gets More Ridiculous By The Day</h6>
+                    <hr>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="col-md-7">
```

Comment:
there are 3 break points in this project (sm,md,lg), please adjust and use them.